### PR TITLE
Migrated to new workflow and publication to GH packages

### PR DESCRIPTION
**Pull request by andreas-borglin**
Example release: https://github.com/AEVI-AppFlow/pos-android-sdk/releases/tag/0.0.2
Example packages: https://github.com/orgs/AEVI-AppFlow/packages?repo_name=pos-android-sdk

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,16 +23,22 @@ Please download the latest developer bundle from [here](https://developer.aevi.c
...
 In your root project `build.gradle` file, add
 
 ```
 repositories {
     maven {
-        url "http://dl.bintray.com/aevi/aevi-uk"
+        name = "AEVI-repos"
+        url = uri("https://maven.pkg.github.com/aevi-appflow/*")
+        credentials {
+            username = <your Github username>
```
Comment (andreas-borglin) [inline README.md:39]: Right, so should we create a GH user and provide details here? Bit indecisive...

Comment (brettwold) [reply README.md:39]: Why would we need to? Surely that's down to the developer using it?

Comment (andreas-borglin) [reply README.md:43]: Yeah but there will be complaints of course as that is a major pain, especially for CI. Not everyone will have a Github "robot" and would then have to use private Github details in CI etc. It's not very user friendly

**Review by brettwold — APPROVED** [review]
Not sure I'm qualified to look at this anymore :)

**Review by xdevl — APPROVED** [review]

**Pull request merged**